tests/components/accessibilityControls-test.js

```diff
@@ -117,8 +117,25 @@ describe('AccessibilityControls', function () {
     };
 
     var accessibilityControls = new AccessibilityControls(controllerMock);
-    accessibilityControls.handleKey(mockEvent);
-    accessibilityControls.handleKey(mockEvent);
+    accessibilityControls.keyEventDown(mockEvent);
+    accessibilityControls.keyEventDown(mockEvent);
+  });
+
+  it('tests "A" key', function () {
```

Comment:
Some of the older unit tests are not very good. I would recommend not basing new ones off of them. Latest changes from master should have a slightly better approach, please use this as a basis.

Reply:
ok, we will, thanks.

Reply:
@jpfranco can you check if the new unit tests are satisfactory? Thanks!